[Feature/main detail] 반응형 및 반복문 코드 수정

dbzero4143 opened the pull request:
[Feature/main detail] 반응형 및 반복문 코드 수정

dbzero4143 committed:
feature/mainDetail

mainDetail UI

dbzero4143 committed:
feature/mainDetail

mainDetail UI

dbzero4143 committed:
feature/mainDetail

mainDetail UI

dbzero4143 committed:
feature/mainDetail

mainDetail UI

dbzero4143 committed:
Fix: Responsive Page & v-for modify

Fix: Responsive Page & v-for modify

dbzero4143 committed:
Fix: Responsive Page & v-for modify

Fix: Responsive Page & v-for modify

dbzero4143 committed:
Fix: Responsive Page & v-for modify

Fix: Responsive Page & v-for modify

dbzero4143 committed:
Fix: Responsive Page & v-for modify

Fix: Responsive Page & v-for modify

devgaram commented:
[Jenkins]
https://test.dev-up.kr/frontend_PR-27/

devgaram commented on src/views/mainDetail/mainDetail.vue:
```diff
@@ -0,0 +1,181 @@
+<template>
+    <div class="divapp">
+        <v-container fluid grid-list-md>
```
width 전체 화면으로 하진 않을거라서 fluid 빼야 될 것 같아요.

devgaram commented on src/views/mainDetail/mainDetail.vue:
```diff
@@ -0,0 +1,181 @@
+    <div class="divapp">
+        <v-container fluid grid-list-md>
+            <v-layout row wrap>
+                <v-flex d-flex xs12 sm12 md12 justify-center="true">
```
xs12 sm12 이거 뭐예요?..적용 안되는 거 같은 데..

devgaram replied:
v-layout, v-flex 가 제가 알기론 3.x부터 v-row, v-col로 대체되서.. 수정하셔야할 듯 싶어요

dbzero4143 committed:
[Feature/mainDetail] 컴포넌트 레이아웃 및 분리 수정

mainDetail 컴포넌트 레이아웃 및 분리 수정

dbzero4143 committed:
[Feature/mainDetail] 컴포넌트 레이아웃 및 분리 수정

mainDetail 컴포넌트 레이아웃 및 분리 수정

dbzero4143 committed:
[Feature/mainDetail] 컴포넌트 레이아웃 및 분리 수정

mainDetail 컴포넌트 레이아웃 및 분리 수정

dbzero4143 committed:
[Feature/mainDetail] 컴포넌트 레이아웃 및 분리 수정

mainDetail 컴포넌트 레이아웃 및 분리 수정

dbzero4143 committed:
[Feature/mainDetail] v-container fluid 삭제

mainDetail 페이지 v-container fluid 삭제

dbzero4143 committed:
[Feature/mainDetail] v-container fluid 삭제

mainDetail 페이지 v-container fluid 삭제

dbzero4143 committed:
[Feature/mainDetail] v-container fluid 삭제

mainDetail 페이지 v-container fluid 삭제

dbzero4143 committed:
[Feature/mainDetail] v-container fluid 삭제

mainDetail 페이지 v-container fluid 삭제

dbzero4143 committed:
[Feature/mainDetail] mainDetail 리뷰 수정

mainDetail
템플릿 글자 데이터 처리 없애기
모바일 레이아웃 수정
댓글부분 컴포넌트 분리
버튼색 수정
이미지 슬라이더 대신 단일 이미지로 수정

dbzero4143 committed:
[Feature/mainDetail] mainDetail 리뷰 수정

mainDetail
템플릿 글자 데이터 처리 없애기
모바일 레이아웃 수정
댓글부분 컴포넌트 분리
버튼색 수정
이미지 슬라이더 대신 단일 이미지로 수정

dbzero4143 committed:
[Feature/mainDetail] mainDetail 리뷰 수정

mainDetail
템플릿 글자 데이터 처리 없애기
모바일 레이아웃 수정
댓글부분 컴포넌트 분리
버튼색 수정
이미지 슬라이더 대신 단일 이미지로 수정

dbzero4143 committed:
[Feature/mainDetail] mainDetail 리뷰 수정

mainDetail
템플릿 글자 데이터 처리 없애기
모바일 레이아웃 수정
댓글부분 컴포넌트 분리
버튼색 수정
이미지 슬라이더 대신 단일 이미지로 수정

devgaram commented on src/router/index.ts:
```diff
@@ -22,6 +22,16 @@ const routes: RouteConfig[] = [
+      {
+        path: "",
+        component: () => import("@/views/mainDetail/mainDetail.vue")
+      }
```
Path  /mainDetail 말고 /detail/:id 이런식으로 바꾸기로한거 수정하시면될것같아요. detail 아니더라도 괜찮은 명사로 바꿔주세요

devgaram commented on src/views/mainDetail/mainDetail.vue:
```diff
@@ -0,0 +1,18 @@
+<template>
+  <mainDetailComponent></mainDetailComponent>
```
Maindetailcomponent 로 분리한 이유가 뭐예요??

dbzero4143 committed:
Merge branch 'feature/mainDetail' of https://github.com/devv-up/frontend into feature/mainDetail

dbzero4143 committed:
[Feature/mainDetail] error 확인

[Feature/mainDetail] error 확인

dbzero4143 committed:
[Feature/mainDetail] error 확인

[Feature/mainDetail] error 확인

dbzero4143 committed:
[Feature/mainDetail] error발생

[Feature/mainDetail] error발생

dbzero4143 committed:
[Feature/mainDetail] error발생

[Feature/mainDetail] error발생

dbzero4143 committed:
[Feature/mainDetail] detail layout 수정

[Feature/mainDetail] detail layout 수정

dbzero4143 committed:
[Feature/mainDetail] detail layout 수정

[Feature/mainDetail] detail layout 수정

dbzero4143 committed:
[Feature/mainDetail] detail layout 수정

[Feature/mainDetail] detail layout 수정

dbzero4143 committed:
[Feature/mainDetail] detail layout 수정

[Feature/mainDetail] detail layout 수정

dbzero4143 committed:
[Feature/mainDetail] detail layout 재수정

[Feature/mainDetail] detail layout 재수정

dbzero4143 committed:
[Feature/mainDetail] detail layout 재수정

[Feature/mainDetail] detail layout 재수정

devgaram commented on src/components/mainDetail/MainDetailContent.vue:
```diff
@@ -0,0 +1,71 @@
+          <p>모임날짜</p>
+        </v-card-text>
+        <v-card-title class="font-weight-CONDENSED headline ma-2">
+          모임내용
```
v-card-text  v-card-title 이거는 v-card랑 같이 쓰는 거라서 지우고 p태그로 사용하는 게 나을 듯 합니다

devgaram commented on src/components/mainDetail/MainDetailContent.vue:
```diff
@@ -0,0 +1,71 @@
+        <span class="justify-start" max-width="210">
+          <v-row>
+            <v-card-text class="font-weight-CONDENSED headline ml-1">
+              <span class="ml-1">주최자</span>
```
얘도 마찬가지로 p태그로 바꿔주세요

devgaram commented on src/store/api/api.ts:
```diff
@@ -0,0 +1,11 @@
+export const apiCall = axios.create({
+  baseURL: "https://ops-test.com/"
+});
+
```
이 파일은 지금 pr에서는 안쓰니깐 삭제해주세요

devgaram commented on src/store/index.ts:
```diff
@@ -3,13 +3,27 @@ import Vuex from "vuex";
+  startDate: StartDate;
+  endDate: EndDate;
+  timeOfDay: TimeOfDay;
+  location: Location;
```
여기 파일에서 수정한것도 다음 pr에서 진행할거니깐 빼주세요

devgaram commented on src/views/index.ts:
```diff
@@ -0,0 +1,2 @@
+export { default as Main } from "./main/index.vue";
+export { default as Error404 } from "./error404/index.vue";
```
이 파일도 안쓰니깐 지워주세요

devgaram commented:
Vuex 관련된 코드는 다음 pr로 하실거니깐 지우시는 게 나을 것 같아요. 그것만 수정하면 된 듯 싶습니다. 리베이스도 한번 해주세요

devgaram reviewed (approved):
수정하고 머지해주세요

dbzero4143 committed:
[Feature/mainDetail] detail layout 재수정

[Feature/mainDetail] detail layout 재수정

dbzero4143 committed:
Merge branch 'feature/mainDetail' of https://github.com/devv-up/frontend into feature/mainDetail

dbzero4143 committed:
[Feature/mainDetail] detail 재재수정

[Feature/mainDetail] detail 재재수정

dbzero4143 committed:
[Feature/mainDetail] detail 재재수정

[Feature/mainDetail] detail 재재수정
+lint
